src/components/Integrations/BigQuery/ControlPanel/AuthorizationForm.test.js

```diff
@@ -1,70 +1,60 @@
 import React from 'react';
-import { render, fireEvent, waitFor, screen } from '@testing-library/react';
+import { render, fireEvent, waitFor, screen, act } from '@testing-library/react';
 import '@testing-library/jest-dom/extend-expect';
 import IntlProvider from '../../../../i18n/DopplerIntlProvider.double-with-ids-as-values';
 import { AuthorizationForm } from './AuthorizationForm';
-
+import user from '@testing-library/user-event';
```

Comment:
You are importing userEvent, but you are not using it!
Could you change fireEvent to userEvent? It's very easy. 

change method: 
`await fireEvent.change(input, { target: { value: tagToAdd1 } });  -> await userEvent.type(input, tagToAdd1);`

click method: 
`fireEvent.click(addButton); -> userEvent.click(addButton);`

Enter method: 
`fireEvent.keyDown(input, { key: 'Enter', code: 'Enter' }); -> userEvent.type(input, '{enter'});`

Reply:
Add spaces
![image](https://user-images.githubusercontent.com/84402180/131347011-4cd4eb04-c2ec-4c36-b158-a15c1dcd2389.png)

Reply:
ok i agree, in this ocation the import was used for test prupose, i wll to remove this import